cmd/containerd/main.go

```diff
@@ -120,11 +120,13 @@ high performance container runtime
 		)
 		switch runtime {
 		case "runc":
+			logrus.Info("containerd: run with runc executor")
```

Comment:
Yeah, maybe wrap logrus and use containerd's own log api, just don't think https://github.com/sirupsen/logrus#formatters would help in this case, we can consider this later.